README.rst

```diff
@@ -2,14 +2,28 @@ VCenter Operator
 =============
 
 The VCenter Operator automatically configures and deploys cinder and nova-compute nodes corresponding to the discovered vcenters and clusters.
-It follows the convention over configuration principle to keep the configuration to a minimum. It relies heavily on a other configmaps being deployed by `openstack-helm <https://github.com/sapcc/openstack-helm>`_  and should be best deployed with it.
+It follows the convention over configuration principle to keep the configuration to a minimum. It relies heavily on a other configmaps and secrets being deployed by `helm-charts/openstack <https://github.com/sapcc/helm-charts/tree/master/openstack>`_  and should be best deployed with it.
+The helm-chart for the vcenter-operator can be found `here <https://github.com/sapcc/helm-charts/tree/master/openstack/vcenter-operator>`_.
+
+
+Brief Overview
+-------------------
+#. Initially retreiving information about the k8s cluster it runs on
+#. Polling the VCenters username and password from `vcenter-operator` secret
```

Comment:
typo: "VCenters" -> "vCenters" also below

I'd write "secret" as "Secret" because that's the k8s object's kind. One could write `k8s Secret` to make it clear.